client/sites-dashboard/components/sites-plan-renew-nag.tsx

```diff
@@ -87,7 +89,10 @@ export const PlanRenewNag = ( { isSiteOwner, plan, checkoutUrl }: PlanRenewProps
 				{ isSiteOwner && (
```

Comment:
Can we remove the `<span>` from this string?

```
__( '<span>%s - Expired</span>' ),
```

There's no reason to include the HTML in the string here.

Reply:
done!